### PR TITLE
[Notifier] Add Mailjet Bridge

### PR DESCRIPTION
**Pull request by jnadaud**
Mailjet notifier bridge: https://github.com/symfony/symfony/pull/41705

### CODE REVIEW

File: notifier.rst [hunk 1]
```diff
@@ -91,6 +92,10 @@ Twilio          ``symfony/twilio-notifier``           ``twilio://SID:TOKEN@defau
     The Iqsms, GatewayApi, Octopush, AllMySms, Clickatell, SpotHit, FakeSms, LightSms, SmsBiuras
     and MessageBird integrations were introduced in Symfony 5.3.
 
+.. versionadded:: 5.4
+
+    The Mailjet integration were introduced in Symfony 5.4.
```
Comment (OskarStark) [inline notifier.rst]:
```suggestion
    The Mailjet integration was introduced in Symfony 5.4.
```

Comment (jnadaud) [reply notifier.rst]: done

**Comment by OskarStark** [comment]
It looks like `from` is mandatory, can you please use user and pass of the dsn then, while I would use `Auth token` as `password` and `from` as user.

Same topic for another bridge: https://github.com/symfony/symfony/pull/41522/files#r644752617

**Comment by javiereguiluz** [comment]
We'll need a rebase here to make it mergeable. Thanks!

**Comment by OskarStark** [comment]
I added it in #15584

**Pull request closed without merging**